iaso/api/org_unit_change_request_configurations/views_mobile.py

```diff
@@ -92,7 +107,10 @@ def list(self, request: Request, *args, **kwargs) -> Response:
             # we have to sort the resulting list manually to keep the pagination working properly.
             queryset = list(
                 chain(
-                    queryset.exclude(org_unit_type__in=non_editable_org_unit_type_ids),
+                    queryset.exclude(
+                        Q(org_unit_type__in=non_editable_org_unit_type_ids)
+                        & Q(type=OrgUnitChangeRequestConfiguration.Type.EDITION)
```

Comment:
Could things go wrong when `include_creation = true`?

AFAIU line `87` would set `queryset.exclude(type=CREATION)` and this line would set `queryset.exclude(type=EDITION)`.

Reply:
Ok